Make error

I am using esl-erlang_22.3.4.9-1 on WSL2.

When I run make I get below error

===> Compiling quicer
===> Analyzing applications...
===> Compiling emqtt_bench
===> Compiling src/emqtt_bench.erl failed
src/emqtt_bench.erl:493: syntax error before: _000

src/emqtt_bench.erl:477: function run/6 undefined

src/emqtt_bench.erl:950: function shard_addr/2 is unused

make: *** [Makefile:12: compile] Error 1

--- a/src/emqtt_bench.erl
+++ b/src/emqtt_bench.erl
@@ -490,7 +490,7 @@ run(Parent, N, PubSub, Opts0, AddrList, HostList) ->
                     true ->
                         [ {min_heap_size, 16}
                         , {min_bin_vheap_size, 16}
-                        , {fullsweep_after, 1_000}
+                        , {fullsweep_after, 1000}
                         ];
                     false ->
                         []

